service/controller/controllercontext/client.go

```diff
@@ -18,6 +17,5 @@ type ContextClientControlPlane struct {
 
 type ContextClientTenantCluster struct {
 	AWS aws.Clients
-	G8s versioned.Interface
-	K8s kubernetes.Interface
+	K8s k8sclient.Interface
```

Comment:
Restructuring this. Now that we have the full metal client collection we just set and use this.